site/package.json

```diff
@@ -8,6 +8,8 @@
     "lint": "next lint",
     "lingui:extract": "NODE_ENV=test lingui extract --clean",
     "lingui:compile": "NODE_ENV=test lingui compile",
+    "sync": "NODE_ENV=test LINGUI_CONFIG=.linguirc_local lingui extract --overwrite && lingui compile",
```

Comment:
lingui.config.ts should be supported out of the box according to the lingui docs. Which would let us use `process.env`

At some point we only need to keep the api key in Github so the Github action can run the sync for us right?
Do you think devs would want to be running this locally @biwano ?

I believe github should let us add secrets to process.env is this correct @ladytrekker ?

Reply:
Yeah, we should do that. But given the order in which lingui looks at the configuration file, the file commited should be lingui.config.js and the override .linguirc, I will make the change.

Reply:
Yes, that is the idea with github actions. 
It may make sense then to upload new translations of the master translation (pseudo-en ?)to translation.io on every merge to staging atomatically. Let's discuss what would be the best workflow with the new integration.

Reply:
> lingui.config.ts should be supported out of the box according to the lingui docs. Which would let us use `process.env`
Definitely, I changed that.
> 
> At some point we only need to keep the api key in Github so the Github action can run the sync for us right? Do you think devs would want to be running this locally @biwano ?
They should not have to.
> 
> I believe github should let us add secrets to process.env is this correct @ladytrekker ?

I made some changes that makes lingui sync if the KLIMA_TRANSLATION_IO_KEY variable is setup.
I also factorized the linguijs config between app and site. I can make it configurable if need be later.